scripts/cfFindIP.sh

```diff
@@ -13,12 +13,27 @@
 #         NOTES: ---
 #        AUTHOR: Morteza Bashsiz (mb), morteza.bashsiz@gmail.com
 #  ORGANIZATION: Linux
-#       CREATED: 01/20/2023 07:36:57 PM
+#       CREATED: 01/24/2023 07:36:57 PM
 #      REVISION:  ---
 #===============================================================================
 
 set -o nounset                                  # Treat unset variables as an error
 
+# Check if 'parallel', 'nmap' and 'bc' packages are installed
+# If they are not,exit the script
+
+if [[ "$(uname)" == "Linux" ]]; then
+    command -v parallel >/dev/null 2>&1 || { echo >&2 "I require 'parallel' but it's not installed. Please install it and try again."; exit 1; }
+    command -v nmap >/dev/null 2>&1 || { echo >&2 "I require 'nmap' but it's not installed. Please install it and try again."; exit 1; }
+    command -v bc >/dev/null 2>&1 || { echo >&2 "I require 'bc' but it's not installed. Please install it and try again."; exit 1; }
```

Comment:
Checking timeout command here is necessary same as the others

Reply:
added check for timeout,nc replaced by sh